setup.py

```diff
@@ -37,7 +36,6 @@
           'kiwipy[rmq]>=0.5.2',
           'enum34; python_version<"3.4"',
```

Comment:
we can also remove the `enum34` dependency now

Reply:
And the `backports.tempfile` as well

Reply:
done.